### PR TITLE
ensure packets are always added to loss list after estimated timeout

### PR DESCRIPTION
**Pull request by birarda**
https://highfidelity.manuscript.com/f/cases/19958/reliable-send-queue-completely-stuck-causing-missing-avatar-URLs-kill-avatar-packets-etc

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/15284/HighFidelity-Beta-PR14436-23857ea.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/15284/HighFidelity-Beta-PR14436-23857ea.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/15284/HighFidelity-Beta-Interface-PR14436-23857ea.dmg

File: libraries/networking/src/udt/SendQueue.cpp [hunk 1]
```diff
@@ -501,12 +502,31 @@ bool SendQueue::isInactive(bool attemptedToSendPacket) {
                 // We think the client is still waiting for data (based on the sequence number gap)
                 // Let's wait either for a response from the client or until the estimated timeout
                 // (plus the sync interval to allow the client to respond) has elapsed
-                auto waitDuration = std::chrono::microseconds(_estimatedTimeout + _syncInterval);
-                
+
+                auto estimatedTimeout = std::chrono::microseconds(_estimatedTimeout);
```
Comment (ZappoMan) [inline libraries/networking/src/udt/SendQueue.cpp]: why are you dropping the `_syncInterval` clause from this new calculation?

Comment (birarda) [reply libraries/networking/src/udt/SendQueue.cpp]: It is no longer used, had been removed from the Connection and CongestionControl classes.

Comment (ZappoMan) [reply libraries/networking/src/udt/SendQueue.cpp]: I guess it looks like we weren't ever setting this syncInterval?

**Comment by qaAdamSmith82** [comment]
Pass @conklin94122  Per a conversation with Dev, this branch of the PR does not need a full retest.  The build passes a basic smoke test.

**Pull request merged**